src/constants.py

```diff
@@ -1,7 +1,7 @@
 from typing import Type
 
 from src.base_model import BaseModel
-from src.implementor.models import SimulationConfiguration
+from src.implementor.models import Run, SimulationConfiguration
 
 # Add classes that should be created as tables to this list
-tables: list[Type[BaseModel]] = [SimulationConfiguration]
+tables: list[Type[BaseModel]] = [SimulationConfiguration, Run]
```

Comment:
```suggestion
tables: list[Type[BaseModel]] = [Run, SimulationConfiguration]
```
(consistency: same order as the imports)

Reply:
Cool idea 💚